templates/legal/ubuntu-pro-service-terms/2015-09-09.md

```diff
@@ -0,0 +1,117 @@
+
+1.  ## Interpretation
+
+    When used in this Agreement, the following terms mean:
+
+    **Affiliate**: a corporate entity that directly or indirectly controls, is controlled by, or is under common control with a party, where "control" means ownership of more than 50% of the outstanding shares or securities representing the right to vote for the election of directors or other managing authority of such corporate entity.
+
+    **Confidential Information**: the terms of this Agreement, and any information identified by the disclosing party as confidential or which the receiving party reasonably ought to know is confidential in light of the nature of the information or the circumstances of its disclosure.
+
+    **Fee(s)**: the amounts payable by Customer for the Services, as set out in the Order.
+
+    **Intellectual Property Rights**: all vested and future rights of copyright and related rights, design rights, database rights, patents, rights to inventions, trade marks and get-up (and goodwill attaching to those trade marks and that get up), domain names, applications for and the right to apply for any of the above, moral rights, goodwill (and the right to sue for passing off and unfair competition), rights in know-how, rights in confidential information, rights in computer software and semiconductor topographies, and any other intellectual or industrial property rights or equivalent forms of protection, whether or not registered or capable of registration, and all renewals and extensions of such rights, whether now known or in future subsisting in any part of the world.
+
+    **Open Source Software**: any software which is distributed under any of the many known variations of licence terms which allow the free distribution and modification of the software's source code or which require all distributors to make such source code freely available upon request, including any contributions or modifications thereto made by such distributor.
+
+    **Order**: If Customer is buying Services directly from Canonical: Customer's order for Services which incorporates this Agreement and identifies Customer's name, contact information, Services, Fees, and Services Term. If Customer is buying Services through Canonical's reseller: Customer's agreement to purchase the Services and pay applicable Fees.
+
+    **Registration Process**: The process under which a Customer registers with Canonical to receive Services purchased through Canonical's reseller.
+
+    **Services**: the services to be performed by Canonical as identified in the Order and described at [www.ubuntu.com/uasd](/legal/ubuntu-advantage-service-description) and/or [www.ubuntu.com/mssd](/legal/managed-services-description) as of the Effective Date.
```

Comment:
Something seems off here, the URL text is a link that mismatches the actual link.